Sources/MessagingPush/MessagingPush+NotificationCenter.swift

```diff
@@ -42,6 +59,24 @@ public extension MessagingPush {
         return false
     }
 
+    func trackMetric(
+        notificationContent: UNNotificationContent,
+        event: Metric,
+        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void
+    ) {
```

Comment:
Should this function go into `MessagingPush` instead of `MessagingPush+NotificationCenter`?

Reply:
Ah, nevermind. I see this is more of a convenient extension. Got it.